controllers/dns_helper.go

```diff
@@ -232,7 +231,7 @@ func (dh *dnsHelper) getLoadBalancedEndpoints(mcgTarget *multicluster.GatewayTar
 			}
 
 			if len(ipValues) > 0 {
-				clusterLbName := strings.ToLower(fmt.Sprintf("%s.%s", cgwTarget.GetShortCode(), lbName))
+				clusterLbName := strings.ToLower(fmt.Sprintf("%s.%s", clusterID, lbName))
```

Comment:
was chatting with @mikenairn on this. I think we could still use the short code based on the target plus the cluster id? This would allow for two gateways on a single cluster sharing a host for example

Reply:
do you mean like: `<clusterID>-<shortcode>.<lbName>`?

Reply:
yeah WDYT? would allow more use cases I think without much overhead